webv2/web.go

```diff
@@ -1657,8 +1657,8 @@ func init() {
 
 // App connects to the web app v2.
 func App() {
-	addr := ":8080"
+	addr := ":4200"
```

Comment:
This might break existing functionality of the UI running at 8080. We should check if there is anywhere else where 8080 is assumed

Reply:
Previously:
1. webv2 UI used to run on 4200 port only since the UI was served by the `ng serve` (Angular Development server).
2. HB http server used to run on port 8080.

Now:
1. The HTTP webserver now serves the UI as well. S

Since the users are _used to_ opening localhost:4200 to open the webV2 UI, I have kept the experience same.


> Please include changes to documentation as required around build and running process

I have currently added comments on how to build here (Line 52) -
https://github.com/cloudspannerecosystem/harbourbridge/pull/388/files#diff-7d422311b9d1565f2d86bf865102cb1fe0531a3e6b62aa6a67fdffc64b173d5a

If you have a different place in mind, I can move it there.